### PR TITLE
configs/rtl8721csm/tc: update defconfig

### PR DESCRIPTION
**Pull request by sunghan-chang**
1. add ',' at end of partition list
 Because of this missing, smartfs mount fails.

2. remove duplicated LIB_ENVPATH to fix below

.config:1045:warning: override: reassigning to symbol LIB_ENVPATH

3. update it through menuconfig

Signed-off-by: sunghan-chang <sh924.chang@samsung.com>

### CODE REVIEW

**Review by jeongchanKim — APPROVED** [review]
Looks good to me.

File: os/board/rtl8721csm/src/component/soc/realtek/amebad/fwlib/ram_hp/rtl8721dhp_app_start.c [hunk 1]
```diff
@@ -1257,6 +1257,47 @@ static void app_driver_call_os_func_init(void)
...
+ * Description:
+ *   Set the IDLE stack to the coloration value and jump into os_start()
+ *
+ ****************************************************************************/
+
+#ifdef CONFIG_STACK_COLORATION
+static void go_os_start(void *pv, unsigned int nbytes)
+{
+	/* Set the IDLE stack to the stack coloration value then jump to
+	 * nx_start().  We take extreme care here because were currently
```
Comment (jeongchanKim) [inline os/board/rtl8721csm/src/component/soc/realtek/amebad/fwlib/ram_hp/rtl8721dhp_app_start.c:1272]: Please change nx_start to os_start.

File: os/board/rtl8721csm/src/component/soc/realtek/amebad/fwlib/ram_hp/rtl8721dhp_app_start.c [hunk 2]
```diff
@@ -1257,6 +1257,47 @@ static void app_driver_call_os_func_init(void)
...
+
+		"1:\n"                      /* Top of the loop */
+		"\tsub  r1, r1, #1\n"       /* R1 nwords-- */
+		"\tcmp  r1, #0\n"           /* Check (nwords == 0) */
+		"\tstr  r2, [r0], #4\n"     /* Save stack color word, increment stackptr */
+		"\tbne  1b\n"               /* Bottom of the loop */
+
+		"2:\n"
+		"\tmov  r14, #0\n"          /* LR = return address (none) */
+		"\tb    os_start\n"         /* Branch to nx_start */
```
Comment (jeongchanKim) [inline os/board/rtl8721csm/src/component/soc/realtek/amebad/fwlib/ram_hp/rtl8721dhp_app_start.c:1296]: Same as above.

**Comment by sunghan-chang** [comment]
This new go_os_start file is verified on artik053/tc, rtl8721csm/tc.

**Review by jeongchanKim — APPROVED** [review]
LGTM

**Review by sangwon03 — APPROVED** [review]
LGTM

**Pull request merged**